scripts/ingest-tarball.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Ingest a tarball containing software, a compatibility layer,
+# or init scripts to the EESSI CVMFS repository, and generate
+# nested catalogs in a separate transaction.
+# This script has to be run on a CVMFS publisher node.
+
+tar_file=$1
```

Comment:
Move this below the `$#` check? Or move the check up?

Reply:
Done!